README.md

```diff
@@ -2,7 +2,7 @@
 # cursor_heap (aka cheap)
 
 The cursor_heap library is adapted from the cursor_heap component in the
-[Heterogeneous Storage Engine](https://github.com/hse-project/hse). 
+[Heterogeneous-Memory Storage Engine](https://github.com/hse-project/hse). 
```

Comment:
Link to the org: https://github.com/hse-project

Reply:
done